src/components/Guide/AppDescription.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function AppDescription() {
+  return (
+    <Description>
+      <p>
+        Take your Time, helps you schedule and categorize your daily activities.
```

Comment:
I think it's great to have a guide / help section in the app! A teeny tiny detail: I think you don't need the comma behind 'Time' ;)